core/src/main/java/io/grpc/ManagedChannel.java

```diff
@@ -104,4 +104,21 @@ public ConnectivityState getState(boolean requestConnection) {
   public void notifyWhenStateChanged(ConnectivityState source, Runnable callback) {
     throw new UnsupportedOperationException("Not implemented");
   }
+
+  /**
+   * Notifies the channel that a reconnect attempt should be made. The implementation may attempt to
```

Comment:
Please also update the wording here. May be remove the first sentence and organize the rest as:
`For subchannels that are in TRANSIENT_FAILURE status, short-circuit the backoff timer and make them reconnect immediatly . May also attempt to invoke {@link NameResolver#refresh}`

Reply:
Done